src/pages/components/typography.mdx

```diff
@@ -23,14 +23,22 @@ components:
 
 ### Code highlight
 
-Use Code to style and format `<code>` elements. You can change text size and background color. Code
-components come in small, medium, and large.
+Use Code to style and format `<code>` elements. Code components come in small, medium, and large.
 
-import Code from '../../examples/typography/Code.tsx';
-import CodeCode from '!!raw-loader!../../examples/typography/Code.tsx';
+import CodeSize from '../../examples/typography/CodeSize.tsx';
+import CodeSizeCode from '!!raw-loader!../../examples/typography/CodeSize.tsx';
 
-<CodeExample code={CodeCode}>
-  <Code />
+<CodeExample code={CodeSizeCode}>
+  <CodeSize />
+</CodeExample>
```

Comment:
Clicking "View code" on this example results in a hard site crash. See https://5f0ce5cbe13983e8523fa327--zendeskgarden.netlify.app/components/typography#lists for details.